Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -8,14 +8,27 @@ final class PledgeCTAContainerView: UIView {
 
   private let vm: PledgeCTAContainerViewViewModelType = PledgeCTAContainerViewViewModel()
 
-  private lazy var amountAndRewardTitleStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var amountAndRewardTitleStackView: UIStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
   private lazy var amountOrRewardLabel: UILabel = { UILabel(frame: .zero) }()
-  private lazy var pledgeCTAbutton: UIButton = {
+  private lazy var pledgeButton: UIButton = {
```

Comment:
Renamed for simplicity

Reply:
This could have been suggested in https://github.com/kickstarter/ios-oss/pull/702. Also don't think this is a good name because you cannot pledge after a project is over. You can only view rewards.